kolibri/core/models.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+
+from kolibri.plugins import registry
+registry.initialize()
```

Comment:
Can this be done using [AppConfig.ready](https://docs.djangoproject.com/en/1.9/ref/applications/#django.apps.AppConfig.ready)?